benchmarks/django-workload/graphite/docker-graphite-statsd/README.md

```diff
@@ -0,0 +1,235 @@
+Then update the root user's profile at: http://localhost:81/admin/auth/user/1/
+
+## Change the Configuration
+
+Read up on Graphite's [post-install tasks](https://graphite.readthedocs.org/en/latest/install.html#post-install-tasks).
+Focus on the [storage-schemas.conf](https://graphite.readthedocs.org/en/latest/config-carbon.html#storage-schemas-conf).
+
+1. Stop the container `docker stop graphite`.
+1. Find the configuration files on the host by inspecting the container `docker inspect graphite`.
+1. Update the desired config files.
+1. Restart the container `docker start graphite`.
+
+**Note**: If you change settings in `/opt/graphite/conf/storage-schemas.conf`
+be sure to delete the old whisper files under `/opt/graphite/storage/whisper/`.
+
+---
+
+**Important:** Ensure your Statsd flush interval is at least as long as the highest-resolution retention.
+For example, if `/opt/statsd/config.js` looks like this.
+
+```
+flushInterval: 10000
+```
+
+Ensure that `storage-schemas.conf` retentions are no finer grained than 10 seconds.
+
+```
+[all]
+pattern = .*
+retentions = 5s:12h # WRONG
+retentions = 10s:12h # OK
+retentions = 60s:12h # OK
+```
+
+## Statsd Admin Management Interface
+
+A management interface (default on port 8126) allows you to manage statsd & retrieve stats.
+
+```sh
+# show all current counters
+echo counters | nc localhost 8126
+```
+
+[More info & additional commands.](https://github.com/statsd/statsd/blob/master/docs/admin_interface.md)
+
+## Secure Grafana
+
+Update the default Grafana admin account. _The default is insecure._
+
+  * username: admin
+  * password: admin
+  * email: admin@localhost
+
+First login at: http://localhost
+Then update the admin user's profile at: http://localhost/admin/users/edit/1
+
+## Connect Grafana to Graphite
+
+Visit http://localhost/datasources/new
+Then configure the Graphite data source with the URL http://localhost:81
+
+## A Note on Volumes
+
+You may find it useful to mount explicit volumes so configs & data can be managed from a known location on the host.
+
+Simply specify the desired volumes when starting the container.
+
+```
+docker run -d\
+ --name graphite\
+ --restart=always\
+ -v /path/to/graphite/configs:/opt/graphite/conf\
+ -v /path/to/graphite/data:/opt/graphite/storage\
+ -v /path/to/statsd:/opt/statsd\
+ hopsoft/graphite-statsd
+```
+
+**Note**: The container will initialize properly if you mount empty volumes at
+          `/opt/graphite`, `/opt/graphite/conf`, `/opt/graphite/storage`, or `/opt/statsd`
+
+## Memcached config
+
+If you want Graphite to use an existing Memcached server, set the following environment variables:
+
+```
+docker run -d\
+ --name graphite\
+ --restart=always\
+ -p 80:80\
+ -p 2003-2004:2003-2004\
+ -p 2023-2024:2023-2024\
+ -p 8125:8125/udp\
+ -p 8126:8126\
+ -e "MEMCACHE_HOST=127.0.0.1:11211"\  # Memcached host(s) comma delimited
+ -e "CACHE_DURATION=60"\              # in seconds
+ hopsoft/graphite-statsd
```

Comment:
Same as the above comment. I think we should change this line to the cloudsuite repository (e.g., cloudsuite/django:graphite-statsd or any other proper one).

Reply:
This image is not explicitly used while running the django image. 
I have retained the original readme from hopsoft just as a reference.

This readme need not be read/used while deploying django workload